proto/rill/runtime/v1/queries.proto

```diff
@@ -605,3 +612,22 @@ message TableRowsRequest {
 message TableRowsResponse {
   repeated google.protobuf.Struct data = 1;
 }
+
+message DownloadLinkRequest {
+  int32 limit = 1;
+  DownloadFormat format = 2;
+  oneof request {
+    MetricsViewToplistRequest metrics_view_toplist_request = 3;
+    MetricsViewRowsRequest metrics_view_rows_request = 4;
+  }
+}
+
+enum DownloadFormat {
+  DOWNLOAD_FORMAT_UNSPECIFIED = 0; // assumes CSV
```

Comment:
Let's not assume CSV, and instead return an error if it's unspecified. It's not a user-facing API, so better to require the client to be explicit